packages/ui/src/common/model/breadcrumbs.ts

```diff
@@ -0,0 +1,11 @@
+export const knownRoutes = [
+  { path: '/profile', breadcrumb: 'My Profile' },
+  { path: '/profile/memberships', breadcrumb: 'My Memberships' },
+  { path: '/working-groups', breadcrumb: 'Working Groups' },
+  { path: '/working-groups/openings', breadcrumb: 'Openings' },
+  { path: '/working-groups/my-applications', breadcrumb: 'My Applications' },
+  { path: '/working-groups/my-roles', breadcrumb: 'My Roles' },
+  { path: '/members', breadcrumb: 'Members' },
+]
+
+export const excludePaths = ['/', '/working-groups/grouppreview']
```

Comment:
Could you also change the path for the working group?

Having in mind that the list of groups is finite and there shouldn't be a conflict with other routes we could have this route:

```
/working-groups/:groupName

so:

/working-groups/membership
/working-groups/storage
etc
```